frontend/packages/data-portal/app/components/Dataset/DatasetAuthors.tsx

```diff
@@ -1,30 +1,27 @@
-import { Fragment, useMemo } from 'react'
+import { ComponentProps, ComponentType, Fragment, useMemo } from 'react'
 
-import { Dataset_Authors } from 'app/__generated__/graphql'
-import { EnvelopeIcon } from 'app/components/icons'
-import { Link } from 'app/components/Link'
+import { AuthorInfo, AuthorLink } from 'app/components/AuthorLink'
 import { cns } from 'app/utils/cns'
 
-export type AuthorInfo = Pick<
-  Dataset_Authors,
-  'name' | 'primary_author_status' | 'corresponding_author_status' | 'email'
->
-
 function getAuthorKey(author: AuthorInfo) {
   return author.name + author.email
 }
 
+const SEPARATOR = `, `
+
 export function DatasetAuthors({
+  AuthorLinkComponent = AuthorLink,
   authors,
   className,
-  separator = ';',
```

Comment:
I don't have the context, but I see the separator changed from `;` to `, `, is there backward compatibility / API versioning related things we need to account for?